storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -45,10 +47,30 @@ type newDataFileSetReaderFn func(
 	opts fs.Options,
 ) (fs.DataFileSetReader, error)
 
+type runType int
+
+const (
```

Comment:
Any reason to move these out into a more shareable location? I'm thinking no because its really only the FS bootstrapper that uses almost the exact same code-path for bootstrappign data vs index right?

Reply:
Although true, this is internal to this bootstrapper - for instance for now the peer bootstrapper will have a fundamentally different path when bootstrapping the index as it needs to call FetchMetadata on the client session, hence it won't need any concept of run type.